MeesGame/Gameplay/GameObjects/Player/Player.cs

```diff
@@ -95,6 +104,11 @@ public void PerformAction(PlayerAction action)
             if (!CanPerformAction(action)) throw new PlayerActionNotAllowedException();
             LastAction = action;
 
+            if (action.IsDirection())
+            {
+                sprite = new SpriteSheet(getAssetNameFromDirection(action.ToDirection()));
```

Comment:
Yes, but the same is true for other GameObject properties such as position and sprite, which are also in Player, so I think it would be consistent to have this code in Player, instead of TimedPlayer.
I actually would like it if all the player directions would be in one spritesheet. Maybe, @jorisBarkema can put them into one spritesheet? Then only the spritesheet index needs to be modified, instead of loading a new spritesheet, which is a lot better performance-wise